adminer/include/adminer.inc.php

```diff
@@ -762,6 +769,8 @@ function dumpHeaders($identifier, $multi_table = false) {
 	* @return bool whether to print default homepage
 	*/
 	function homepage() {
+		global $adminer;
+		echo (support("scheme") && $_GET['ns'] === '' ? "<p>" . lang('This database contains ' . $adminer->count_schemas() . ' schemas.') . "</p>\n" : "");
```

Comment:
This is not translatable. Use `%d` instead. Add any translations to `lang/xx.inc.php`.